rocky/assets/js/dropdown.js

```diff
@@ -1,30 +1,45 @@
 const dropdowns = document.querySelectorAll(".dropdown");
+let activeDropdown = null; // To track the currently active dropdown
+
+const closeActiveDropdown = () => {
+  if (activeDropdown) {
+    const activeButton = activeDropdown.querySelector(".dropdown-button");
+    const activeList = activeDropdown.querySelector(".dropdown-list");
```

Comment:
Instead of an `activeDropdown` variable to keep track of this, you could consider querying by `aria-expanded`, i.e.

```js
const activeButton = document.querySelector(".dropdown-button[aria-expanded=true]");
```

And instead of randomly selecting "some" `.dropdown-list`, perhaps it would make sense to get the target's `id` from the trigger's `aria-controls` attribute and use that to find it instead, i.e.

```js
const activeList = document.getElementById(activeButton.getAttribute("aria-controls"));
```